apps/judicial-system/backend/src/app/modules/court/court.service.ts

```diff
@@ -44,7 +44,7 @@ export const subTypes: SubTypes = {
   MAJOR_ASSAULT: 'Líkamsárás - meiriháttar',
   MINOR_ASSAULT: 'Líkamsárás - minniháttar',
   AGGRAVATED_ASSAULT: 'Líkamsáras - sérlega hættuleg',
```

Comment:
Please fix this as well. I checked the Auður API and in all cases the correct text is "Líkamsárás...".